qa/zcash/updatecheck.py

```diff
@@ -126,10 +126,7 @@ class Version(list):
     def __init__(self, version_tuple):
         for part in version_tuple:
             if part: # skip None's which can come from optional regexp groups
-                if str(part).isdigit():
-                    self.append(int(part))
-                else:
-                    self.append(part)
+                self.append(int(part)) if str(part).isdigit() else self.append(part)
```

Comment:
If you want to use ternary, this would be clearer:
```suggestion
                self.append(int(part) if str(part).isdigit() else part)
```